src/tools/csg_dlptopol.cc

```diff
@@ -245,7 +245,7 @@ void DLPTopolApp::WriteMoleculeInteractions(ostream &out, Molecule &cg) {
   votca::Index n_entries = 0;
   votca::Index nb = -1;
 
-  for (Interaction *ic : cg.Interactions()) {
+  for (const auto &ic : cg.Interactions()) {
```

Comment:
ic is not a good name, especially with the `auto`

Reply:
but it is common in csg. So this is a nitpick

Reply:
Actually I think ic is short for interaction container so I should really change it.